continuous-pipe.yml.erb

```diff
@@ -5,6 +5,9 @@ variables:
     name: CP_ENVIRONMENT
     expression: '"<%= config.name %>" ~ code_reference.branch'
   CP_ENVIRONMENT_DOMAIN_SUFFIX: -<%= config.name %>.webpipeline.net
+  SENDMAIL_RELAY_HOST:
+    name: SENDMAIL_RELAY_HOST
+    expression: '"mailcatcher.<%= config.name %>-" ~ slugify(code_reference.branch) ~ ".svc.cluster.local"'
```

Comment:
actually, I just remembered, as I've been trying it, that variables aren't available within variables, only env vars